AppController/lib/cassandra_helper.rb

```diff
@@ -17,14 +17,18 @@
 
 
 # A String that indicates where we install Cassandra on this machine.
-CASSANDRA_DIR = "#{APPSCALE_HOME}/AppDB/cassandra"
+CASSANDRA_DIR = "/opt/cassandra"
```

Comment:
Is this dir persisted in cloud mode?
I think it should be something like /opt/appscale/cassandra-env to make sure we store in external volumes.